devices/heiman.js

```diff
@@ -672,4 +672,19 @@ module.exports = [
         },
         exposes: [e.battery(), e.action(['pressed']), e.battery_low(), e.tamper()],
     },
+    {
+        fingerprint: [{modelID: 'SOS-EF-3.0', manufacturerName: 'HEIMAN'}],
+        model: 'SOS-EF-3.0',
```

Comment:
Is there any model number written on the site/product page?

Reply:
It´s an HS1EB
[https://de.aliexpress.com/item/4000075282903.html](url)

Reply:
Then please merge it with https://github.com/dominkhense/zigbee-herdsman-converters/blob/c680e0a0f5c41c9f4c96d84545a25b0c24cb9bb5/devices/heiman.js#L402